### ISSUE TITLE
Add some needed functionality to feature/CalendarPageRedesign

### ISSUE THREAD

**Issue by Caleb-Cohen** [Caleb-Cohen/Together#245]
## Please provide a summary of the feature

@luciousmc did a beautiful calendar page redesign. Currently there are several buttons in the nav button that have not been completed. These must be completed before we pull into development. 

- Add a button to the nav bar that has the functionality to return to landing page.
- Have the add event modal trigger on button press (Pending #157).
- Remove add event form to the bottom of the login.
- Add feedback button functionality. I'd love a modal popup that allows users to type in their feedback to submit, but that may be a little much to start now. It could redirect directly to the issues tab on github so users can post their feedback/issus.
- Add help button functionality. Can be just a modal popup with a link to the discord group thread, and github issues tab.

While all these issues must be completed before the issue is closed. You can pickup specific ones and PR just those fixes to `feature/CalendarPageRedesign`

**When Working on these you have to continue the work of the `feature/CalendarPageRedesign` branch not `development`**

## Is this feature related to any current or past issues? Please provide the #

#157 will need to be completed prior to the functionality added to the button.

## Which Branch should this fix be Pulled into?
PR into feature/CalendarPageRedesign from feature/CalendarPageRedesign

*Starting out of development will not have the old Calendar Page*

*If you're interested in working on this feature, please comment so you can be assigned*

**Comment by luciousmc** [comment]
I wanted to give some time for someone else if they wanted to pick this up, but I can be the point person for it. x

> Add a button to the nav bar that has the functionality to return to landing page.

@Natashka333 for the button would a house icon suffice? Or would it be completely something else?

> Remove add event form to the bottom of the login.

What do you mean by this? @Caleb-Cohen 

> Add feedback button functionality. I'd love a modal popup that allows users to type in their feedback to submit, but that may be a little much to start now. It could redirect directly to the issues tab on github so users can post their feedback/issus.

This would involve adding a new collection to the db for feedback.  Something like...

```
{
  id: 'blahblah',
  user: userid,
  feedback: 'this site so cool thx'
}
```
or should we do it another way?

> Add help button functionality. Can be just a modal popup with a link to the discord group thread, and github issues tab.

Currently I have the `Join Team` button linking to the discord group thread. What would you like that to do instead? @Caleb-Cohen

**Comment by Caleb-Cohen** [comment]
- House button is ok for me
- The add event form no longer needs to be at the bottom of the form because the add event form was turned into a modal #157 and should be triggered by the add event button. I may need to do some github magic to get these features together.
<img width="948" alt="Screenshot 2023-01-13 at 5 17 43 PM" src="https://user-images.githubusercontent.com/75268174/212436037-94f0f990-6d56-4836-ab21-d2608e579403.png">
- I do not think the feedback should start a new collection. It's a good idea and can be implemented in the future, but for time and complexity reasons I would prefer if it just linked to the issues tab on github and people can provide feedback if any there.
- Join team and help can lead to the same place for now.

In terms of functionality I'd prefer the simplest approach that does not add complexity, if possible. We can always break down these features in the future and improve them!

**Comment by Natashka333** [comment]
@luciousmc, I think the home button should be good.
Kinda like this?
![Screen Shot 2023-01-13 at 5 31 07 PM](https://user-images.githubusercontent.com/102708144/212437181-f8de23b0-2985-4947-ba12-49686f7e659d.png)

**Comment by luciousmc** [comment]
>  I do not think the feedback should start a new collection. 

How will we store the feedback? I wouldn’t think it add it to the user model. I also don’t think it’d add much complexity. I’ll link it to the issues tab for now.

Once the add event modal is merged I can complete the issues concerning it.
Home button and other link functions can be completed as soon as I get home tonight.

**Comment by Caleb-Cohen** [comment]
> >  I do not think the feedback should start a new collection. 
> 
> 
> 
> How will we store the feedback? I wouldn’t think it add it to the user model. I also don’t think it’d add much complexity. I’ll link it to the issues tab for now.
> 
> 
> 
> Once the add event modal is merged I can complete the issues concerning it.
> 
> Home button and other link functions can be completed as soon as I get home tonight.

The feedback would hyperlink to issues tab on GitHub so people can post their feedback there. There would not be a in-app functionality for feedback in this release.

**Comment by luciousmc** [comment]
Oh ok, I remember you saying at the standup that you wanted it in so I skimmed over the part above where you stated not now. My bad!

**Comment by Caleb-Cohen** [comment]
> Oh ok, I remember you saying at the standup that you wanted it in so I skimmed over the part above where you stated not now. My bad!

Nah, that's my bad. I want it, and it's part of the design. Just with multiple redesigns coming in at the same time, I would prefer to get this feature shipped around the same time even if that means adding full functionality at a later date.
